axelor-stock/src/main/java/com/axelor/apps/stock/service/StockMoveServiceImpl.java

```diff
@@ -373,6 +380,13 @@ public String realize(StockMove stockMove) throws AxelorException {
   @Transactional(rollbackOn = {Exception.class})
   public String realize(StockMove stockMove, boolean checkOngoingInventoryFlag)
```

Comment:
Please remove the previously added check for stock move realization in StockMoveController

Reply:
I remove the check in StockMoveController in the first commit of the PR.